### ISSUE TITLE
Can the program be transplanted to ESP32 to run?

### ISSUE THREAD

**Issue by changxinchang** [harlock974/time-signal#2]
Can the program be transplanted to ESP32? It is hoped that NTP time can be obtained through ESP32 networking, and then analog radio signals can be sent. Thank you.

**Comment by harlock974** [comment]
Yes but it needs a major rewrite of clock-control.c, as this part of the program is Raspberry Pi dependent.

For ESP32 NTP please have a look of my other repository : https://framagit.org/harlock/esp8266-ntp-clock

**Comment by changxinchang** [comment]
Hello, can you update a version that supports ESP32? Only touched arduino ESP32. Can't use raspberry pie.

**Comment by changxinchang** [comment]
I don't know the code of raspberry pie. I hope I can understand the code based on arduino ESP32 and add the support of BPC radio waves. Thanks again

**Comment by harlock974** [comment]
Hello, I don't plan to rewrite the code for other platforms for now.
Anyway there are already similar projects done for ESP32 :
https://www.startpage.com/do/dsearch?q=DCF77+transmitter+ESP32&cat=web
